.github/DISCUSSION_TEMPLATE/Q&A.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Standard Q&A template
+about: List a question for the community of WAM2layers
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Welcome to WAM2layers! 
+
+If you're new here, please check our [readme](https://github.com/WAM2layers/WAM2layers), 
+[documentation](https://wam2layers.readthedocs.io/en/latest/) first. 
+      
+Still questions? Please introduce yourself if you are a new user, and please ask your question as clear as possible. We hope the community gets back to you a.s.a.p.
```

Comment:
```suggestion
Still have questions? Please introduce yourself if you are a new user, and ask your question as clearly as possible. We try to keep an eye on this forum and we hope that everyone who benefits from it will do the same, so help can come as quickly as possible.
```